src/services/proof-of-good-smart-contract.service.ts

```diff
@@ -167,6 +167,34 @@ export class ProofOfGoodSmartContractService {
       return this.contract.setCap(activityId, oracleId, duration, points);
     });
   }
+
+  async getUserGoodPoints(userId: string) {
+    try {
+      const res = await this.contract.profileByUserId(userId);
+      const [
+        profileId,
+        walletAddresses,
+        balance,
+        totalGood,
+        categories,
+        entries,
+      ] = res;
```

Comment:
```suggestion
      const [
        profileId,
        walletAddresses,
        balance,
        totalGood,
        categories,
        entries,
      ] = await this.contract.profileByUserId(userId);
```
we don't use res elsewhere :)

Reply:
good catch